src/webgpu/api/validation/createBindGroupLayout.spec.ts

```diff
@@ -1,7 +1,7 @@
 export const description = `
 createBindGroupLayout validation tests.
 
-TODO: update for new binding structure, remove eslint-disable.
+TODO: update for new binding structure.
```

Comment:
I just realized this TODO should be obsolete, I did it when fixing deprecations.